web/store/issue/workspace/filter.store.ts

```diff
@@ -213,11 +211,6 @@ export class WorkspaceIssuesFilter extends IssueFilterHelperStore implements IWo
             _filters.displayFilters.group_by = "state";
             updatedDisplayFilters.group_by = "state";
           }
-          // set sub_issue to false if layout is switched to spreadsheet and sub_issue is true
-          if (_filters.displayFilters.layout === "spreadsheet" && _filters.displayFilters.sub_issue === true) {
```

Comment:
Can you please help me understand why this was added in the first place, since you were the original author